Form-Controls/styles.css

```diff
@@ -0,0 +1,127 @@
+:root {
+    --white:#FFF;
+    --black:#000;
+    --db:#E6DACD;
```

Comment:
Always avoid this kind of cryptic shorthand. We write code for humans to read. (We will learn how to compress front end code later on, in something called an asset pipeline.)

https://syllabus.codeyourfuture.io/guides/code-style-guide#avoid-short-names